src/main/java/frc/robot/Robot.java

```diff
@@ -215,6 +238,29 @@ public CommandBase camToggle() {
         return camTogglecmd;
     }
 
+    public CommandBase camOn() {
+        CommandBase cmd = new InstantCommand(() -> {
+            SmartDashboard.putBoolean("Is Shooting", true);
+        });
+        cmd.setName("Camera On");
+        return cmd;
+    }
+
+    public CommandBase camOff() {
+        CommandBase cmd = new InstantCommand(() -> {
+            SmartDashboard.putBoolean("Is Shooting", false);
+        });
+        cmd.setName("Camera Off");
+        return cmd;
+    }
+
+    public CommandBase visionAlignment() {
+        CommandBase cmd = new SequentialCommandGroup(camToggle(), led.visionGreenOn(), drive.visionAlignDegrees(),
```

Comment:
What is it toggling here?

Reply:
We did have Cam on, I thought I changed it guess not